apple/RNCWebView.m

```diff
@@ -472,12 +513,32 @@ - (void)setAllowsBackForwardNavigationGestures:(BOOL)allowsBackForwardNavigation
 }
 
 - (void)removeFromSuperview
+{
+  bool keepWebViewInstance = _webViewKey != nil;
+  if (!keepWebViewInstance) {
+    [self cleanUpWebView];
+  }
+  
+  if (_webViewKey != nil) {
```

Comment:
I made it an if-else, since one of them is checking for `shouldReuseWebView`, and the other is checking for `!shouldReuseWebView`. https://github.com/react-native-webview/react-native-webview/commit/829967139d6ee111a8269f0d6e3e7a5fa68b60b6